data/org.gnome.shell.gschema.xml.in

```diff
@@ -126,16 +126,6 @@
         Whether the application grid displays the App Center icon.
       </description>
     </key>
-    <key name="enable-code-view" type="b">
```

Comment:
I think you should squash it with the commit "data: Add hack-mode-enabled setting"